### PR TITLE
This is the codeblock

### PR DESCRIPTION
**Pull request by GrizzlyScar**

### CODE REVIEW

**Review by giancarlopernudisegura — CHANGES_REQUESTED** [review]
I'm getting lots of `Module Not Found` errors. Make sure that the libraries that you're using are in the `package.json` so everything works after running `yarn install`.

**Comment by GrizzlyScar** [comment]
Can you try the latest commit and see if it works?

File: examples/pages/_app.tsx [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import "@neuralbertatech/ui/style.css";
 
 import type { AppProps } from "next/app";
+import 'highlight.js/styles/hybrid.css';
+import '../styles/globals.css'
```
Comment (giancarlopernudisegura) [inline examples/pages/_app.tsx:5]: You've included some css import but not the css file.
```
Module not found: Can't resolve '../styles/globals.css'
  3 | import type { AppProps } from "next/app";
  4 | import 'highlight.js/styles/hybrid.css';
> 5 | import '../styles/globals.css'
  6 | 
  7 | function MyApp({ Component, pageProps }: AppProps) {
  8 |   return <Component {...pageProps} />;
```

**Review by giancarlopernudisegura — CHANGES_REQUESTED** [review]

**Comment by giancarlopernudisegura** [comment]
Thanks for the submission. I'm going to make some changes before merging the PR.

**Review by giancarlopernudisegura — APPROVED** [review]

**Pull request merged**